community_images/httpd/bitnami/functional_test.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -x
+set -e
+
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# shellcheck disable=SC1091
+. "${SCRIPTPATH}"/../../common/helpers.sh
+
+HELM_RELEASE=apache-release
+NAMESPACE=$(get_namespace_string "${HELM_RELEASE}")
+REPOSITORY=apache
+IMAGE_REPOSITORY=rapidfort/"$REPOSITORY"
+
+k8s_test()
```

Comment:
we should also add some tests to k8s_test as well. probably the coverage script can be added here as well to begin with.

Reply:
Noted.